dotnet/src/SemanticKernel.Core/KernelExtensions.cs

```diff
@@ -446,5 +447,146 @@ public static ISKPlugin ImportPluginFromObject(this Kernel kernel, object target
 
         return kernel.RunAsync(function, variables ?? new(), cancellationToken);
     }
+
+    /// <summary>
+    /// Run a pipeline composed of synchronous and asynchronous functions.
+    /// </summary>
+    /// <param name="kernel">The kernel.</param>
+    /// <param name="variables">Input to process</param>
+    /// <param name="cancellationToken">The <see cref="CancellationToken"/> to monitor for cancellation requests. The default is <see cref="CancellationToken.None"/>.</param>
+    /// <param name="pipeline">List of functions</param>
+    /// <returns>Result of the function composition</returns>
+    /// <inheritdoc/>
+    public static async Task<FunctionResult> RunAsync(this Kernel kernel, ContextVariables variables, CancellationToken cancellationToken, params KernelFunction[] pipeline)
```

Comment:
Isn't the RunAsync method to run a pipeline method gone?

Reply:
Good catch, this might be kept by the merge. Will remove